pelican/plugins/sitemap/sitemap.py

```diff
@@ -36,14 +36,11 @@
 """
 
 
-def format_date(date):
-    """Format the date in the expected format."""
-    if date.tzinfo:
-        tz = date.strftime("%z")
-        tz = tz[:-2] + ":" + tz[-2:]
-    else:
-        tz = "-00:00"
-    return date.strftime("%Y-%m-%dT%H:%M:%S") + tz
+def format_date(date_time):
+    """Format the datetime in the expected format."""
+    if date_time.tzinfo is None:
+        date_time = date_time.replace(tzinfo=timezone.utc)
```

Comment:
Does replacing tzinfo do any time conversion, or is naive datetime always UTC and not localtime?

Reply:
Naive datetime has no time zone. The old implementation did set an invalid UTC timezone `-00:00` when no time zone was present. I just fixed that invalid UTC timezone.

Please check docs on .replace(): https://docs.python.org/3/library/datetime.html#datetime.datetime.replace

When you change a time zone with replace,  it just changes timezone nothing else.